docs: In App Support (#396)
frappe/erpnext_com: erpnext_com/www/in-app-support.md

<section class='top-section'>
<h1>In-App Support</h1>
</section>

> To serve Customers more efficiently on Support Issues, Frappe Technologies is implementing an In-App Support allowing you to raise and track issues right from your erpnext account.

Reviewer: ```suggestion
To serve Customers more efficiently on Support Issues, Frappe Technologies is implementing an In-App Support allowing you to raise and track issues right from your ERPNext account.
```

#### Using In App Support

> In-App Support is accessible in the `Help` dropdown. Click on `ERPNext Support` and follow the steps to raise a Support Ticket

Reviewer: ```suggestion
In-App Support is accessible in the **Help** dropdown. Click on **ERPNext Support** and follow the following steps to raise a Support Ticket
```

- Creating a new Support Issue using the In-App Support.
<img class="img-responsive feature-image screenshot" src="/assets/erpnext_com/images/in-app-support/sp.gif">

- All the Communication that takes place between You and Frappe Technologies is maintained in the respective Support Issue.
<img class="img-responsive feature-image screenshot" src="/assets/erpnext_com/images/in-app-support/sp2.gif">

> - You can reply to an Issue by click on the `Reply` button in the timeline.

Reviewer: ```suggestion
- You can reply to an Issue by click on the **Reply** button in the timeline.
```

<img class="img-responsive feature-image screenshot" src="/assets/erpnext_com/images/in-app-support/sp3.gif">

> - An Issue created is visible to all the Users of your Company in order to reduce duplicate Issue reporting.

Reviewer: ```suggestion
- An Issue created by a user is visible to all the users of your ERPNext account in order to reduce duplicate Issue reporting.
```

> <img class="img-responsive feature-image screenshot" src="/assets/erpnext_com/images/in-app-support/sp1.png">

Reviewer: crop top and bottom
